# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/windows/ShellHeaderOnlyUtils.h
# Commit: abcbfdf1a1ef
# Full Hash: abcbfdf1a1efad1d852db7889ebda328c4e8b73d
# Author: Toshihito Kikuchi <tkikuchi@mozilla.com>
# Date: 2019-11-15 09:53:19
# Regressor Bug: 1567614
# File Overlap Count: 1
# Description:
#   Bug 1588975 - Replace ShellExecuteExW with mozilla::ShellExecuteByExplorer. r=aklotz,asuth
#   
#   The launcher process turns on the `PreferSystem32Images` mitigation policy for
#   the browser process.  Since the mitigation policy is inherited, a process launched
#   by the browser process also has `PreferSystem32Images`.  If an application which
# ==============================================================================

diff -r 36bef03d2b69 -r abcbfdf1a1ef widget/windows/ShellHeaderOnlyUtils.h
--- a/widget/windows/ShellHeaderOnlyUtils.h	Thu Nov 14 18:36:28 2019 +0000
+++ b/widget/windows/ShellHeaderOnlyUtils.h	Thu Nov 14 19:47:29 2019 +0000
@@ -140,7 +140,7 @@
 #endif  // defined(ShellExecute)
 
   // 4. Now call IShellDispatch2::ShellExecute to ask Explorer to execute.
-  hr = shellDisp->ShellExecute(aPath, aArgs, aVerb, aWorkingDir, aShowCmd);
+  hr = shellDisp->ShellExecute(aPath, aArgs, aWorkingDir, aVerb, aShowCmd);
   if (FAILED(hr)) {
     return LAUNCHER_ERROR_FROM_HRESULT(hr);
   }